internal/compiler/tests/syntax/elements/text.slint

```diff
@@ -0,0 +1,13 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+export component Foo inherits Rectangle {
+    Text {
+        font-metrics: 42;
+//      ^error{Cannot assign to output property 'font-metrics'}
+//                    ^^error{Cannot convert float to slint::private_api::FontMetrics}        
```

Comment:
The name of this type in this error is not ideal. But i guess this is not the only type with this problem.

Reply:
Right. I'm also not too happy about this, but this should rarely happen :)